samples/Sentry.Samples.Maui/MainPage.xaml.cs

```diff
@@ -27,6 +29,28 @@ protected override void OnAppearing()
         base.OnAppearing();
     }
 
+    private void AddHandlerBreadcrumbs()
+    {
+        foreach (var visualElement in this.GetVisualTreeDescendants())
+        {
+            if (visualElement is Element element)
+            {
+                element.HandlerChanged += (sender, _) =>
```

Comment:
We're subscribing to events here but not unsubscribing anywhere. This is often a memory leak source.

Here's some tips on looking for leaks: https://github.com/dotnet/maui/wiki/Memory-Leaks